### ISSUE TITLE
Invoking `catkin config` (no arguments) clears out Make Args

### ISSUE THREAD

**Issue by tprk77** [catkin/catkin_tools#225]
I am using `0.3.0` installed from `pip`.

I can save `make` arguments using `catkin config ...` as expected, but if I try to read the config using `catkin config` (no arguments) the make arguments get cleared out. CMake arguments are not affected, if that helps. For example:

```text
tperkins@bluedragon:~/test_ws$ catkin config --make-args -j1 -l1 -- --cmake-args -DWHATEVER=1 --
-----------------------------------------------------------------
Profile:                     default
Extending:                   None
Workspace:                   /data/users/tperkins/test_ws
Source Space:       [exists] /data/users/tperkins/test_ws/src
Build Space:        [exists] /data/users/tperkins/test_ws/build
Devel Space:        [exists] /data/users/tperkins/test_ws/devel
Install Space:      [exists] /data/users/tperkins/test_ws/install
DESTDIR:                     None
-----------------------------------------------------------------
Isolate Develspaces:         False
Install Packages:            False
Isolate Installs:            False
-----------------------------------------------------------------
Additional CMake Args:       -DWHATEVER=1
Additional Make Args:        -j1 -l1
Additional catkin Make Args: None
Internal Make Job Server:    True
-----------------------------------------------------------------
Whitelisted Packages:        None
Blacklisted Packages:        None
-----------------------------------------------------------------
Workspace configuration appears valid.
-----------------------------------------------------------------
tperkins@bluedragon:~/test_ws$ cat .catkin_tools/default/config.yaml 
blacklist: []
build_space: build
catkin_make_args: []
cmake_args:
- -DWHATEVER=1
devel_space: devel
extend_path: null
install: false
install_space: install
isolate_devel: false
isolate_install: false
make_args:
- -j1
- -l1
source_space: src
use_internal_make_jobserver: true
whitelist: []
tperkins@bluedragon:~/test_ws$ catkin config
-----------------------------------------------------------------
Profile:                     default
Extending:                   None
Workspace:                   /data/users/tperkins/test_ws
Source Space:       [exists] /data/users/tperkins/test_ws/src
Build Space:        [exists] /data/users/tperkins/test_ws/build
Devel Space:        [exists] /data/users/tperkins/test_ws/devel
Install Space:      [exists] /data/users/tperkins/test_ws/install
DESTDIR:                     None
-----------------------------------------------------------------
Isolate Develspaces:         False
Install Packages:            False
Isolate Installs:            False
-----------------------------------------------------------------
Additional CMake Args:       -DWHATEVER=1
Additional Make Args:        None
Additional catkin Make Args: None
Internal Make Job Server:    True
-----------------------------------------------------------------
Whitelisted Packages:        None
Blacklisted Packages:        None
-----------------------------------------------------------------
Workspace configuration appears valid.
-----------------------------------------------------------------
tperkins@bluedragon:~/test_ws$ cat .catkin_tools/default/config.yaml 
blacklist: []
build_space: build
catkin_make_args: []
cmake_args:
- -DWHATEVER=1
devel_space: devel
extend_path: null
install: false
install_space: install
isolate_devel: false
isolate_install: false
make_args: []
source_space: src
use_internal_make_jobserver: true
whitelist: []
```

**Comment by wjwwood** [comment]
Is this a duplicate of https://github.com/catkin/catkin_tools/issues/136

**Comment by tprk77** [comment]
I think it is unrelated. Or possibly a regression? I checked my source to make sure I have commit 4007bcdf6e2d1b140fdda733b1a6580854d4a196, which fixed #136. I think I have commit 1cacc064f374930f3f3fa0ea89e9ddbc5c95c196.

**Comment by tprk77** [comment]
Ok, I checked it out. I definitely have 1cacc064f374930f3f3fa0ea89e9ddbc5c95c196.

**Comment by jbohren** [comment]
Closed with #276
